webclient/src/context/itin/plan-reducer.js

```diff
@@ -0,0 +1,80 @@
+import { actionTypes } from './plan-actions';
+
+export const initPlan = {
+  itin: localStorage.getItem('plan-info')
+    ? JSON.parse(localStorage.getItem('plan-info')).itin
+    : {
+        // intro: [
+        //   {
+        //     _id: 0,
+        //     name: 'Introduction to NUS',
+        //     description: 'Lorem ipsum si dolor amet'
+        //   }
+        // ]
+      }
+};
+
+export const planReducer = (state = initPlan, action) => {

```

Comment:
hmm what do you think of initPlan to be a function instead? perhaps more suitable to retrieve the latest value in localStorage?